tracer/samples/Samples.Transport.UnixDomainSocket/Dockerfile

```diff
@@ -0,0 +1,134 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS appbase
+ARG DD_API_KEY
+ENV DD_API_KEY=$DD_API_KEY
+
+EXPOSE 80
+EXPOSE 443
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/runtime-deps:5.0-buster-slim as builder
+ARG DOTNETSDK_VERSION=6.0.100
```

Comment:
This was a copy paste of the debian dockerfile we use for CI (which now might be stale)... I wonder if I could include a script to pre-build that dockerfile and use a tag here instead of duplicating?